File: Iniciante/URI 1010.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

var line1 = lines.shift().split(' ');
var cod1 = parseInt(line1[0]);
var n1 = parseInt(line1[1]);
var valor1 = parseFloat(line1[2]);

var line2 = lines.shift().split(' ');
var cod2 = parseInt(line2[0]);
var n2 = parseInt(line2[1]);
var valor2 = parseFloat(line2[2]);

var total = (n1 * valor1) + (n2 * valor2);
console.log('VALOR A PAGAR: R$ ' + total.toFixed(2));

File: Iniciante/URI 1010.dart
import 'dart:io';

void main() {
  List<String> line1 = stdin.readLineSync().split(" ");
  int cod1 = int.parse(line1[0]);
  int n1 = int.parse(line1[1]);
  double valor1 = double.parse(line1[2]);

  List<String> line2 = stdin.readLineSync().split(" ");
  int cod2 = int.parse(line2[0]);
  int n2 = int.parse(line2[1]);
  double valor2 = double.parse(line2[2]);

  double total = (n1 * valor1) + (n2 * valor2);
  print("VALOR A PAGAR: R\$ ${total.toStringAsFixed(2)}");
}